packages/fast-components-react-msft/src/select/select.stories.tsx

```diff
@@ -154,12 +158,14 @@ storiesOf("Select", module)
             />
         </Select>
     ))
-    .add("Adjacent", () => (
+    .add("Right", () => (
         <Select
             placeholder="Select an option"
             onValueChange={action("onValueChange")}
             menuFlyoutConfig={{
                 horizontalPositioningMode: AxisPositioningMode.adjacent,
```

Comment:
does this mean that since it is locked to default `horizontalPositioningMode` does nothing?

Reply:
As it's implemented now the "defaults" are the ones allowed by the current positioning mode settings.  I could see setting it up so that locking to a default would override the other settings but that would be a task unto itself.